doc/sphinx/user-extensions/proof-schemes.rst

```diff
@@ -197,8 +200,15 @@ Combined Scheme
 Generation of inversion principles with ``Derive`` ``Inversion``
 -----------------------------------------------------------------
 
-.. cmd:: Derive Inversion @ident with @ident Sort @sort
-         Derive Inversion @ident with (forall {* @binder }, @ident @term) Sort @sort
+.. cmd:: Derive Inversion @ident with @one_term {? Sort @sort_family }
+
+   .. insertprodn sort_family sort_family
+
+   .. prodn::
+      sort_family ::= Set
+      | Prop
+      | SProp
+      | Type
 
    This command generates an inversion principle for the
    :tacn:`inversion ... using ...` tactic.  The first :token:`ident` is the name
```

Comment:
This description is now incorrect. There should be a description of what `one_term` can be.